### PR TITLE
Update deployment config for enableModel

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: deploy/overlays/stanford-staging/kustomization.yaml [hunk 1]
```diff
@@ -19,29 +19,29 @@ nameSuffix: -stanford-staging
 namespace: website
 
 resources:
-- ../../base
```
Comment (Fructokinase) [inline deploy/overlays/stanford-staging/kustomization.yaml:22]: is this intended?

Comment (Fructokinase) [reply deploy/overlays/stanford-staging/kustomization.yaml:22]: For my own learning, are these the same?

```
resources:
- A


resources:
  - A
```

**Review by Fructokinase — APPROVED** [review]

File: deploy/overlays/stanford-staging/kustomization.yaml (hunk 1 above)
Comment (shifucun) [reply deploy/overlays/stanford-staging/kustomization.yaml:22]: Yes, which we used in other files. I get this formatting from vscode linter tool.

**Review by shifucun — COMMENTED** [review]
thanks!

**Pull request merged**